src/strategies/MetamaskStrategy.ts

```diff
@@ -54,6 +54,10 @@ export class MetamaskStrategy implements WalletStrategy {
     }
   }
 
+  async openSettings() {
```

Comment:
The Metamask don't have this method so please throw `Error` to notice users

Reply:
Added an error to the metamask strategy.
Feel free to change to a more appropriate message if needed.

Reply:
Sorry for the late reply.
It's looking good!